external/include/headers/spirv.h

```diff
@@ -50,12 +50,12 @@
 
 typedef unsigned int SpvId;
 
-#define SPV_VERSION 100
-#define SPV_REVISION 1
+#define SPV_VERSION 10000
```

Comment:
Right... It's interesting to see that `SpvVersion` (which we actually use) is defined correctly.